client/me/purchases/remove-purchase/index.jsx

```diff
@@ -29,9 +45,35 @@ const RemovePurchase = React.createClass( {
 	},
 
 	getInitialState() {
+		// shuffle reason order, but keep another_reason_one last
+		const questionOneOrder = shuffle( [
+			'could_not_install',
+			'too_hard',
+			'did_not_include',
+			'only_need_free'
+		] );
+		questionOneOrder.push( 'another_reason_one' );
+
+		const questionTwoOrder = shuffle( [
+			'staying_here',
+			'other_wordpress',
+			'different_service',
+			'no_need'
+		] );
+		questionTwoOrder.push( 'another_reason_two' );
+
 		return {
 			isDialogVisible: false,
-			isRemoving: false
+			isSurveyOneVisible: config.isEnabled( 'upgrades/removal-survey' ),
+			isSurveyTwoVisible: ! config.isEnabled( 'upgrades/removal-survey' ),
```

Comment:
It's a workaround to hide the first part of the survey to users which don't have `config.isEnabled( 'upgrades/removal-survey' )`. SurveyTwo is essentially the original Dialog for the `RemovePurchase` flow with an extra text field (that is hidden if 'upgrades/removal-survey' isn't enabled).